### PR TITLE
Matches Hipchat's notification format

### PR DESCRIPTION
**Pull request by jbabe**
🐘 

amaabca/capistrano-amadeploy#17
- PR above will make our version of capistrano-deploy match the Hipchat
  notification format in Capistrano (since we'll be moving to regular
  Capistrano)
- This matches this new pattern
- update tests to match

### CODE REVIEW

**Comment by jbabe** [comment]
👋

**Comment by mvandenbeuken** [comment]
🐘 🐘 🐘

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

**Comment by jbabe** [comment]
💩 

I think I should update this to match both the old and new patterns so we don't have to do a sweep of all the old apps (meaning I can close https://github.com/amaabca/capistrano-amadeploy/pull/17)

**Comment by jbabe** [comment]
I'll be changing this to match both patterns (cap2 and cap3)

**Comment by darkodosenovic** [comment]
:wave:

File: lib/lita/handlers/server_status.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 module Lita
   module Handlers
     class ServerStatus < Handler
-      MESSAGE_REGEX = /(.+) is starting deploy of '(.+)' from branch '(.+)' to (.+)/i
-
+      MESSAGE_REGEX = /(.+) is deploying (.+)\/(.+) to (.+)(\s\(.*)/i
```
Comment (MathieuGilbert) [inline lib/lita/handlers/server_status.rb]: This can just be `(.+) is deploying (.+)\/(.+) to (.+)\s.*` to not capture the `(production)` part at the end.

**Comment by MathieuGilbert** [comment]
💩

**Comment by darkodosenovic** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Comment by darkodosenovic** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**